src/js/modules/articles/reducer.js

```diff
@@ -22,36 +18,21 @@ const initialState = {
 
 const reducer = (state = { ...initialState }, action) => {
   switch (action.type) {
-    case FETCH_ARTICLES_PENDING: {
-      return { ...state, isFetching: true };
-    }
-    case FETCH_ARTICLES_FULFILLED: {
-      const newArticles = action.payload.reduce((acc, article) => {
-        article["originalSummary"] = article["summary"];
-        article["summary"] = shortenSummary(article);
-        acc[article.id] = article;
-        return acc;
-      }, {});
-      return {
-        ...state,
-        isFetching: false,
-        isFetched: state.authorships.length !== 0,
-        articles: newArticles,
-      };
-    }
-    case FETCH_ARTICLES_REJECTED: {
+    case FETCH_INIT_DATA_FULFILLED: {
       return {
         ...state,
-        isFetching: false,
-        error: action.payload,
-      };
-    }
-    case FETCH_AUTHORSHIPS_FULFILLED: {
-      return {
-        ...state,
-        isFetched: !isObjectEmpty(state.articles),
-        authorships: action.payload,
-      };
+        articles: action.payload.articles.reduce((acc, article) => {
+          article["originalSummary"] = article["summary"];
```

Comment:
nitpicking here but why originalSummary? Why not just article.shortenedSummary

Reply:
Because I expect the summary to be already fixed when I access it in my components. shortenSummary, as the comment says, basically "fixes" my summary.